jmclient/jmclient/wallet_rpc.py

```diff
@@ -424,14 +433,26 @@ def dummy_restart_callback(msg):
                         token=self.cookie)
 
     def taker_finished(self, res, fromtx=False, waittime=0.0, txdetails=None):
```

Comment:
I guess it would have been just as reasonable to make a `tumbler_taker_finished` or similar, and set that as callback for that case (in `start_tumbler`) but this way works well.

Of course it brings up that whole thing - the 'sendpayment vs tumbler' distinction is to my mind no longer the right distinction to make - there are only coinjoin schedules at the lower layer, with an offshoot being the `direct_send` which is an entirely different animal. Your choice of endpoint name `schedule` goes along with that.

I could argue this is perhaps an error; if in future we want the API to support any arbitrary schedule, then it would naturally fit under `/schedule`, but this endpoint implicitly uses the tumbler options. Not sure if that matters right now.

Reply:
I agree completely on the point about 'sendpayment vs tumbler'. I feel like mid-term this endpoint could evolve into simply taking any schedule via POST body and then executing it. For people who would want to use the default tumbler schedule, we could provide another endpoint `GET /schedule` that takes the tumbler options as input and spits out a schedule (which can then subsequently be posted to `POST /schedule`. 

For now though I feel like the all-in-one function as it is now is probably the best (the easiest) way to start.